gtsfm/scene_optimizer.py

```diff
@@ -52,8 +56,8 @@
 MVS_PLY_SAVE_FPATH = RESULTS_PATH / "mvs_output" / "dense_pointcloud.ply"
 
 # Paths to Save Output in React Folders.
-REACT_METRICS_PATH = Path(__file__).resolve().parent.parent / "rtf_vis_tool" / "src" / "result_metrics"
-REACT_RESULTS_PATH = Path(__file__).resolve().parent.parent / "rtf_vis_tool" / "public" / "results"
+REACT_METRICS_PATH = BASE_PATH / "rtf_vis_tool" / "src" / "result_metrics"
+REACT_RESULTS_PATH = BASE_PATH / "rtf_vis_tool" / "public" / "results"
```

Comment:
can we remove this altogether?

Reply:
I think Frank was talking about some student who will pick up the JS work again. So lets keep it for the time being.

Reply:
Yeah, but isnt it the same data that we save in both results_path and react_metrics_path ? Do we need this duplication?

Reply:
I remember there was some need for this because the viz tool could not access the parent of the root folder.